docs/infrastructure_and_maintenance/security/security_checklist.md

```diff
@@ -203,19 +144,69 @@ Reduce your attack surface by exposing only what you must.
             - { path: ^/search, roles: ROLE_USER}
     ```
 
-## Underlying stack
+## Symfony
 
-Once you have properly configured secure user roles and permissions, to avoid exposing your application to any DDOS vulnerabilities or other yet unknown security threats, make sure that you do the following:
+### `APP_SECRET`
 
-- Avoid exposing servers on the open internet when not strictly required.
-- Ensure any servers, services, ports and virtual hosts that were opened for testing purposes are shut down before going live.
-- Ensure file system permissions are set up in such a way that the web server or PHP user can't access files they shouldn't be able to read.
-- Secure the database with a good password, keys, firewall.
-Optionally, ensure that the database user used by the web app only has permissions to do the operations needed by [[= product_name =]].
-The Data Definition Language (DDL) commands (create, alter, drop, truncate, comment) are only needed for installing and upgrading [[= product_name =]], and not for running it.
-Not granting these rights to web app users reduces the damage that can result from a security breach.
+`APP_SECRET` needs to be a strong, random, securely stored value.
 
-Those steps aren't needed when using [[= product_name_cloud =]], where the provider handles them.
+- Don't use a default value like `ff6dc61a329dc96652bb092ec58981f7` or `ThisTokenIsNotSoSecretChangeIt`.
+- The secret must be secured against unwanted access. Don't commit the value to a version control system.
```

Comment:
Maybe something about `.env.local` containing all the secrets, passwords, and sensible data, even `.env.local` added to `.gitignore`.

Reply:
Good stuff. Also, I'll need to modify the related "Secure secrets" section.

Reply:
```suggestion
- The secret must be secured against unwanted access. Don't commit the value to a version control system. There are several ways of handling it, like with enviroment variables or files like `.env.local`. Files are considered more secure. If you store the secrets in files, make sure to add those files to `.gitignore` or similar, so they will never be committed to version control systems.
```

Reply:
Merged "Secure secrets" into this section: https://github.com/ibexa/documentation-developer/pull/2565/commits/6941f052cab1fe9d8f56c551a56a2feae940ff4d